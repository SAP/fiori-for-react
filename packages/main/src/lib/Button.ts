import { Button, ButtonPropTypes } from '../webComponents/Button';

export { Button, ButtonPropTypes };
