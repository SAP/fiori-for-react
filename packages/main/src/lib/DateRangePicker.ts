import { DateRangePicker } from '../webComponents/DateRangePicker';

export { DateRangePicker };
