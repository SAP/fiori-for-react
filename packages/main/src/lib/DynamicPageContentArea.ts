import { DynamicPageContentArea } from '../components/DynamicPageContentArea';

export { DynamicPageContentArea };
