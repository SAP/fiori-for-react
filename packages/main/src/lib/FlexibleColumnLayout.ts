import { FlexibleColumnLayout } from '../webComponents/FlexibleColumnLayout';

export { FlexibleColumnLayout };
