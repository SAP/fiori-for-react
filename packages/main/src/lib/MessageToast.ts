import { MessageToast } from '../components/MessageToast';

export { MessageToast };

// Required for React lazy
export default MessageToast;
