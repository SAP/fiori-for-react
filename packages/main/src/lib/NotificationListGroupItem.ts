import { NotificationListGroupItem } from '../webComponents/NotificationListGroupItem';

export { NotificationListGroupItem };
