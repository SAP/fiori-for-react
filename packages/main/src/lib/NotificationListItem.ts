import { NotificationListItem } from '../webComponents/NotificationListItem';

export { NotificationListItem };
