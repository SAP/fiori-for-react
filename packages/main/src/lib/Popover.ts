import { Popover } from '../webComponents/Popover/popover_refs';

export { Popover };
