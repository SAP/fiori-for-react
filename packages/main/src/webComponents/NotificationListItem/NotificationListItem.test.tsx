import { render } from '@shared/tests';
import { NotificationListItem } from '@ui5/webcomponents-react/lib/NotificationListItem';
import React from 'react';

describe('NotificationListItem', () => {
  test('Basic Test (generated)', () => {
    const { asFragment } = render(<NotificationListItem />);
    expect(asFragment()).toMatchSnapshot();
  });
});
