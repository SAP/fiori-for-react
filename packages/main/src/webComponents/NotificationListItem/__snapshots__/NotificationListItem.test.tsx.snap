// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationListItem Basic Test (generated) 1`] = `
<DocumentFragment>
  <ui5-li-notification
    priority="None"
    wrap="false"
  />
</DocumentFragment>
`;
